addons/l10n_it_stock_ddt/data/l10n_it_ddt_template.xml

```diff
@@ -1,20 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <odoo>
      <template id="my_view_name" inherit_id="l10n_it_edi.account_invoice_it_FatturaPA_export">
-            <xpath expr='//DatiDDT' position="after">
-                <t t-if="ddt_dict and not record.l10n_it_ddt_id">
-                    <t t-foreach="ddt_dict" t-as="picking">
-                        <DatiDDT>
-                            <NumeroDDT t-esc="picking.l10n_it_ddt_number"/>
-                            <DataDDT t-esc="format_date(picking.date)"/>
-                            <t t-if="len(ddt_dict) > 1">
-                                <t t-foreach="ddt_dict[picking]" t-as="line_ref">
-                                    <RiferimentoNumeroLinea t-esc="line_ref"/>
-                                </t>
-                            </t>
-                        </DatiDDT>
-                    </t>
+        <xpath expr='//DatiDDT' position="after">
```

Comment:
Indentation fix for easier reading and troubleshooting in the output XML that goes to the Tax Agency